docker/Dockerfile

```diff
@@ -159,12 +159,28 @@ RUN echo "trusted = true" >> pg_ivm.control && \
     make clean -j && \
     make USE_PGXS=1 -j
 
+######################
+# Barman Cloud
+######################
+FROM postgres:${PG_VERSION_MAJOR}-bookworm AS barman-cloud-base
```

Comment:
If you are installing this directly, there's no point in having a separate docker stage. Let's just combine this as an extra step in the final image?

Otherwise if you could please handle the CI failures and it looks good to me :)